sdk/storage/tests.yml

```diff
@@ -50,7 +51,4 @@ extends:
         TEST_MODE: 'RunLiveNoRecord'
```

Comment:
Definitely some fat to be trimmed in the lines above ^^ but wanting to check CI status first in this least-changed state